libraries/botbuilder-azure/src/index.ts

```diff
@@ -8,3 +8,4 @@
 export * from './cosmosDbStorage';
 export * from './blobStorage';
 export * from './azureBlobTranscriptStore';
+export * from './cosmosDbKeyEscape';
```

Comment:
Is this something you want devs to be able to call?  If so ignore the @private comments and flush out the documentation

Reply:
Spoke with @cleemullins, this implementation mirrors the .NET SDK in 
 that [`CosmosDBKeyEscape.EscapeKey`](https://github.com/Microsoft/botbuilder-dotnet/blob/master/libraries/Microsoft.Bot.Builder.Azure/CosmosDBKeyEscape.cs#L31) is public so it can be covered by tests.

For the JS side of things we don't really need it to be public to still test it, but it makes sense from a C# POV.